examples/aws/terraform/ha-autoscale-cluster/ssm.tf

```diff
@@ -3,17 +3,8 @@
 // is destroyed, cluster will overwrite them with real values
 
 resource "aws_ssm_parameter" "license" {
-  count     = var.license_path != "" ? 1 : 0
-  name      = "/teleport/${var.cluster_name}/license"
-  type      = "SecureString"
-  value     = file(var.license_path)
-  overwrite = true
```

Comment:
Why was `overwrite` removed here?

Reply:
Ah, good catch. It is deprecated, but it's not removed until v6.0.0 of the provider. I just misread the deprecation notice.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_parameter states:

> `overwrite` also makes it possible to overwrite an existing SSM Parameter that's not created by Terraform before. This argument has been deprecated and will be removed in v6.0.0 of the provider. For more information on how this affects the behavior of this resource, see [this issue comment](https://github.com/hashicorp/terraform-provider-aws/issues/25636#issuecomment-1623661159).

The default in v6.0.0 will be `true`, but until then, we do still need this.

Reply:
Cool, TIL!